terraspace/app/modules/lambda-from-sqs/variables.tf

```diff
@@ -3,10 +3,10 @@ variable "sqs_trigger" {
     arn                                = string
     batch_size                         = number
     maximum_batching_window_in_seconds = number
+    function_response_types            = "ReportBatchItemFailures"
```

Comment:
Not sure if this is right. If possible, I want users of this module to not have to provide a value for this at all, and if none is proivded, the value should be tf `null`. If it is provided, it has to be this string value.

If it's not possible to express that.... then I probably need to update all the other uses of this `lambda-from-sqs` module to pass in `function_response_types = null`. But I didn't see any lint/plan errors that required that.... I could be missing something

Reply:
You set a value where it expects a type. That's the error it throws: (I saw the "plan" workflow has thrown a different exception, so I'm starting to think you can get more productive by running plans locally). 

│ Error: Invalid type specification
│ 
│   on ../../modules/lambda-from-sqs/variables.tf line 6, in variable "sqs_trigger":
│    6:     function_response_types            = "ReportBatchItemFailures"
│ 
│ A type specification is either a primitive type keyword (bool, number,
│ string) or a complex type constructor call, like list(string).

Reply:
You can achieve what you're trying to do be using the "default" keyword for a variable. Read this: https://www.terraform.io/language/values/variables

But you're not going be able use that functionality in a type object variable (unless you set all attributes with default values, and that's not what we want do do here) so it might be better to create a different variable for this use case.

Reply:
@francardoso93 thanks. I added a separate variable with default https://github.com/elastic-ipfs/infrastructure/pull/156/commits/a05a0ad81d3c1f57dba1be4f3d8e267fc6020117